<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupSetDlg</class>
 <widget class="BaseDlg" name="StartupSetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1550</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StartupSetDlg</string>
  </property>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>1257</x>
     <y>817</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>1387</x>
     <y>817</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewStartupSet">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>1450</width>
     <height>660</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTimeEdit  {
	height: 35px;
width: 100px;

}
QTableView::item  {
	padding: 0 10px;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>180</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QFrame" name="line_h">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>1450</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>1</width>
     <height>660</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_v_2">
   <property name="geometry">
    <rect>
     <x>1499</x>
     <y>130</y>
     <width>1</width>
     <height>660</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_h_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>789</y>
     <width>1450</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lines" stdset="0">
    <string notr="true">solid</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>871</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="lebalAutoStart">
      <property name="text">
       <string>统一时间设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:url(:/Leonis/resource/image/temp_arrow.png) no-repeat center right;</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="accelerated">
       <bool>true</bool>
      </property>
      <property name="displayFormat">
       <string>HH:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>StartupSetDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
